app/collins/models/LshwHelper.scala

```diff
@@ -86,6 +92,41 @@ object LshwHelper extends CommonHelper[LshwRepresentation] {
     )
   }
 
+  protected def reconstructGpu(meta: Map[Int, Seq[MetaWrapper]]): FilteredSeq[Gpu] = {
```

Comment:
Yeah, I modeled `reconstructGpu` and `collectGpus` off of the existing CPU/memory/disk/NIC functions. I wasn't sure how consistent you'd want the new GPU code to be compared to the previous CPU/memory/disk/NIC stuff.